examples/tls_server-rs/ta/src/main.rs

```diff
@@ -131,36 +144,44 @@ pub fn do_tls_write(session_id: u32, buf: &mut [u8]) -> usize {
     rc
 }
 
-fn make_config() -> Arc<rustls::ServerConfig> {
-    let certs = load_certs();
-    let privkey = load_private_key();
-    let config = rustls::ServerConfig::builder()
-        .with_safe_defaults()
+fn make_config() -> anyhow::Result<Arc<rustls::ServerConfig>> {
+    trace_println!("[+] Creating crypto provider");
+    let crypto_provider = Arc::new(rustls_provider::optee_crypto_provider());
+
+    trace_println!("[+] Creating time provider");
+    let time_provider = Arc::new(rustls_provider::optee_time_provider());
+
+    let certs = load_certs().context("Failed to load certificates")?;
+    trace_println!("[+] Loaded {} certificates", certs.len());
+
+    let private_key = load_private_key().context("Failed to load private key")?;
+    trace_println!(
```

Comment:
Private data should never be printed inside Trusted Applications (TAs), especially in demonstration examples.